mozci/push.py

```diff
@@ -192,7 +192,12 @@ def add(data):
         # artifacts.
         # TODO: We have fallbacks for groups and results, but not for kind.
         try:
-            add(run_query('push_tasks_from_unittest', args)['data'])
+            add(run_query('push_tasks_results_from_unittest', args)['data'])
+        except MissingDataError:
+            pass
+
+        try:
+            add(run_query('push_tasks_groups_from_unittest', args)['data'])
```

Comment:
good splitting.  I wish the `run.suite.groups` was on the `task` table, but `{"eq":{"etl.id":0}}` is as good as we can do for now.